src/flicking.js

```diff
@@ -93,7 +93,7 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		construct: function (element, options, _prefix) {
 			this.$wrapper = $(element);
 
-			if (!this.$wrapper[0] || !this.$wrapper[0].hasChildNodes()) {
+			if (!this.$wrapper[0] || !this.$wrapper.children().length) {

```

Comment:
Great job!
Could you improve more, not repeatedly calling `.children()` on `._build()`?

```js
_build: function () {
       ...
	var children = panel.$list = this.$wrapper.children();
```